README.md

```diff
@@ -1 +1,196 @@
+
+## 🛠 Built With <a name="built-with"></a>
+  -CSS3
+  -HTML5
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Version Control</summary>
+  <ul>
+    <li><a href="https://github.com/">Github</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Visual Studio Code</summary>
+  <ul>
+    <li><a href="https://code.visualstudiocode.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Linters_Setup]**
+- **[Follow_HTMLandCSS_best_practice]**
+- **[Display_Text]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://github.com/TiteLunda/Hello-world)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+-Visual Code Studio
+-Github Account
+-GitBash
+-Node Js
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Use git clone command or download ZIP folder
+
+
+### Install
+
+Install this project with:
+
+npm
+
+
+### Usage
+
+To run the project, execute the following command:
+
+-npm  start or live Server
+
+
+### Run tests
+
+To run tests, run the following command:
+
+npm test
+
+
+
+### Deployment
+
+You can deploy this project using:
+
+-Github Pages
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Tite Lunda**
+
+- GitHub: [@TiteLunda](https://github.com/TiteLunda)
+- Twitter: [@LundaTite](https://twitter.com/LundaTite)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tite-lunda-094956199/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Responsiveness]**
+- [ ] **[Javascript_Developments]**
+- [ ] **[Screen_Resolution]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+If you like this project, you can follow me on Github, linkdin or Twitter for more .
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to thank My team Partners and Microverse staff
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] you did a great job 😃 at adding the MIT LICENCE, but I see that when clicking on the link to open the page of LICENCE I see that it is redirecting me to the page not found, so please 😊 try to put the correct link for the MIT LICENCE to let anyone when clicking on the link of LICENCE redirect him/her to the page of the LICENCE 💯